config/defaults.js

```diff
@@ -150,10 +150,12 @@ module.exports = {
     password:   process.env.PULSE_PASSWORD
   },
 
-  // Statsd configuration options (these are totally optional).
-  statsd: {
-    prefix: process.env.STATSD_PREFIX || '',
-    url: process.env.STATSD_URL || 'tcp://localhost:8125'
+  influx: {
+    connectionString: undefined,
+    maxDelay: 10,
+    maxPendingPoints: 100,
+    // Only collect host level metrics every minute
+    hostMetricsInterval: 60 * 1000
```

Comment:
I'm tempted to suggest that we put this else where... all the other config options here fit right into `base.stats.Influx` constructor...